main.go

```diff
@@ -67,14 +67,26 @@ var (
 			return fmt.Sprintf("\x1b[;38;5;%dm%s\x1b[0m", code+232, s)
 		}
 	}
+
+	// Flag variables
+	onlyHeader bool
 )
 
+func init() {
+	flag.BoolVar(&onlyHeader, "head", false, "show only header of response")
```

Comment:
It's the same flag:
` -I, --head          Show document info only`
But I can name it `-I`